pkgs/by-name/ne/netmaker-ui/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, nix-update-script
+, ...
+}:
+buildNpmPackage rec {
+  pname = "netmaker-ui";
+
+  passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch=master" ]; };
+  version = "unstable-2024-01-22";
+
+  src = fetchFromGitHub {
+    owner = "nazarewk";
+    repo = "netmaker-ui-2";
```

Comment:
Why are we using fork and not the official repository ?

Reply:
added note in a description